self-core-impl/src/main/java/com/selfxdsd/core/contributors/ProjectContributors.java

```diff
@@ -180,6 +178,10 @@ public Contributor elect(final Task task) {
                                 ).divide(
                                     BigDecimal.valueOf(60),
                                     RoundingMode.HALF_UP
+                                ).add(task
```

Comment:
@amihaiemil the above if condition shouldn't be: `(contract.role().equals(task.role()) && contract.project().equals(task.project())` ?

Reply:
@criske yes, we could add that check too. Can you write a puzzle for it? 
Also, in this case, I think it's easier if you say:

```
.add(
    this.project.projectManager().commission()
)
```